jgrapht-core/src/main/java/org/jgrapht/alg/connectivity/GabowStrongConnectivityInspector.java

```diff
@@ -31,20 +31,20 @@
  * @param <E> the graph edge type
  *
  * @author Sarah Komla-Ebri
+ * @author Hannes Wellmann
  */
 public class GabowStrongConnectivityInspector<V, E>
     extends
     AbstractStrongConnectivityInspector<V, E>
 {
-    // stores the vertices
-    private Deque<VertexNumber<V>> stack = new ArrayDeque<>();
+    // the sequence of (original) vertices encountered but not yet assigned to a component
+    private Deque<VertexNumber<V>> S = new ArrayDeque<>();
```

Comment:
Since our coding conventions use initLower for non-static data members, let's call these stackS and stackB (that way you can still keep the connection to Gabow's notation).

(I realize there are other places where we violate the convention; we should probably use checkstyle to clean up and enforce those.)

Reply:
Totally agree. I just didn't want to change too much.

I vote +1 for the idea to enforce it with checkstyle.

What also could be cleaned up and enforced with checkstyle is the use of braces for control-structures (if, for, while, do).
I see it quite often in JGraphT that the braces for a control-structure's body is skipped if the body is a one-liner. From all I know, this is considered bad-style because it is error prone.